clang/test/CIR/CodeGen/ternary.cpp

```diff
@@ -9,7 +9,7 @@ int x(int y) {
 // CHECK:     %0 = cir.alloca !s32i, !cir.ptr<!s32i>, ["y", init] {alignment = 4 : i64}
 // CHECK:     %1 = cir.alloca !s32i, !cir.ptr<!s32i>, ["__retval"] {alignment = 4 : i64}
 // CHECK:     cir.store %arg0, %0 : !s32i, !cir.ptr<!s32i>
-// CHECK:     %2 = cir.load %0 : !cir.ptr<!s32i>, !s32i
+// CHECK:     %2 = cir.load tbaa({{.*}}) %0 : !cir.ptr<!s32i>, !s32i
```

Comment:
Follow up from previous comment: after you add the alias we should filecheck the alias, even if it's empty for now, instead of `{{.*}}` here and elsewhere.